decentraland/unity-renderer
# feat: sdk7 ui input submit

* Added new `NavigationSubmitEvent` to `UiInput`'s `TextField` `VisualElement` to output a `UiInpuResult` component with the new `isSubmit` bool in `true`. Allowed keys for submitting are only Return/Enter and keypad's Return/Enter.
* Added textfield clearing on submit
* Fixed and re-enabled some Ui component tests including `UiInput`

**NOTE:** The PR is actually quite small, had many file changes due to some test assemblies I had to update.

Related PRs
* https://github.com/decentraland/protocol/pull/182
* https://github.com/decentraland/js-sdk-toolchain/pull/809

--------------------------------

### QA TEST INSTRUCTIONS

1. Enter [this playground link](https://playground.decentraland.org/?explorer-branch=feat%2Fsdk7-ui-input-submit&sdk-branch=feat%2Fui-input-submit&code=CgpSZWFjdEVjc1JlbmRlcmVyLnNldFVpUmVuZGVyZXIoKCkgPT4gKAogIDxVaUVudGl0eQogICAgdWlUcmFuc2Zvcm09e3sKICAgICAgd2lkdGg6IDQwMCwKICAgICAgaGVpZ2h0OiAzMDAsCiAgICAgIHBvc2l0aW9uVHlwZTogImFic29sdXRlIiwKICAgICAgcG9zaXRpb246IHsKICAgICAgICBsZWZ0OiAiMzUlIiwKICAgICAgICB0b3A6ICI0MCUiCiAgICAgIH0KICAgIH19CiAgICB1aUJhY2tncm91bmQ9e3sKICAgICAgY29sb3I6IENvbG9yNC5HcmF5KCkKICAgIH19CiAgPgogICAgPElucHV0CiAgICAgICAgb25TdWJtaXQ9eyAodmFsdWUpID0%2BIHsKICAgICAgICAgIGNvbnNvbGUubG9nKCJzdWJtaXR0ZWQgdmFsdWU6ICIgKyB2YWx1ZSkKICAgICAgICB9fQogICAgICAgIGZvbnRTaXplPXszNX0KICAgICAgICBwbGFjZWhvbGRlcj17InR5cGUgc29tZXRoaW5nIn0KICAgICAgICBwbGFjZWhvbGRlckNvbG9yPXtDb2xvcjQuQmxhY2soKX0KICAgICAgICB1aVRyYW5zZm9ybT17ewogICAgICAgICAgd2lkdGg6ICI0MDBweCIsCiAgICAgICAgICBoZWlnaHQ6ICI4MHB4IiwKICAgICAgICB9fQogICAgPgogICAgPC9JbnB1dD4KICA8L1VpRW50aXR5PgopKQo%3D)
2. Wait until the world loads and hit the RUN button
3. Write something in the text input and hit ENTER/RETURN
4. Open the browser console (F12) and check that what you wrote is displayed in a console log
